packages/server/api/src/app/flags/flag.module.ts

```diff
@@ -18,6 +18,10 @@ export const flagController: FastifyPluginAsyncTypebox = async (app) => {
             logLevel: 'silent',
         },
         async (request: FastifyRequest) => {
+            // Retrieve the release candidate version from environment variables
+            const releaseCandidateVersion = process.env.RELEASE_CANDIDATE_VERSION || 'unknown';
```

Comment:
Please move it to `flag.service` and create new entry in `flag.ts` look for `ApFlagId.CURRENT_VERSION`.

You should read the the rc version from the package.json like `getCurrentRelease()`